src/components/WalletSelector/walletUtils.ts

```diff
@@ -122,8 +123,8 @@ export async function listenForGnosisSignature(
   const listenToGnosisSafeContract = new Promise((resolve) => {
     gnosisSafeContract.on(GNOSIS_SAFE_SIGN_MESSAGE_EVENT_NAME, async (msgHash: any) => {
       // Upon detecing the SignMsg event, validate that the contract signed the message
-      const magicValue = await gnosisSafeContract.isValidSignature(messageHash, '0x');
-      const messageWasSigned = magicValue === GNOSIS_VALID_SIGNATURE_MAGIC_VALUE;
+      console.log('Gnosis Safe Contract Event: ', GNOSIS_SAFE_SIGN_MESSAGE_EVENT_NAME);
```

Comment:
do we want to keep this?

Reply:
I thought it might be useful to be able to see that the SignMsg event was detected - otherwise we can't really tell that the event was detected which has slowed down debugging - though do we remove console.logs on other envs?